src/testup/editor.rb

```diff
@@ -74,8 +74,9 @@ def self.open_file(file, line = 0)
     arguments = self.arguments
     arguments.gsub!('{FILE}', file)
     arguments.gsub!('{LINE}', line)
-    command = %["#{editor}" #{arguments}]
-    system(command)
+    # Do not occupy the current process
+    arr = arguments.gsub("\"","").split(" ")
+    Kernel.spawn(editor,*arr)
```

Comment:
This might leave zombie processes:

https://ruby-doc.org/core-2.5.5/Kernel.html#method-i-spawn

> The parent process should use Process.wait to collect the termination status of its child or use Process.detach to register disinterest in their status; otherwise, the operating system may accumulate zombie processes.

So this should be:

```ruby
id = Kernel.spawn(editor,*arr)
Process.detach(id)
```

Reply:
Thank you, I will fix it.